core/src/main/java/io/aiven/klaw/controller/UsersTeamsController.java

```diff
@@ -115,23 +111,6 @@ public ResponseEntity<ApiResponse> registerUser(@Valid @RequestBody RegisterUser
         usersTeamsControllerService.registerUser(newUser, true), HttpStatus.OK);
   }
 
-  @PostMapping(
-      value = "/registerUserSaas",
-      produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<ApiResponse> registerUserSaas(
-      @Valid @RequestBody RegisterSaasUserInfoModel newUser) throws Exception {
-    return new ResponseEntity<>(saasService.registerUserSaas(newUser), HttpStatus.OK);
-  }
-
-  @RequestMapping(
-      value = "/getActivationInfo",
```

Comment:
There are a few calls that still call getActivationInfo, 

registerUsers.js has a call out to the api, which is called from userActivation.html

But I don't know if this is strictly saas related. it could also be used for signup

Reply:
Yes, I felt weird deleting this one! But I think it had dependencies that relied on a bunch of SAAS things 🤔

Reply:
Ok, it's becauyse it's defined on the `SaasService` class. Should it be moved to another class? I don't think we should have a `SaasService` file if there is no SAAS :o

Reply:
I agree we shouldn't have a saasService unless we support saas let me take a look at the code again and get back to you

Reply:
@muralibasani maybe you can double check, but I don't see the userActivation.html page used/called anywhere, I think that html page can also be deleted?

Reply:
@aindriu-aiven so after the user is activated from email through signup process, he is redirected to this. Looks like we need this.

Reply:
@muralibasani had another look.
After Registration the superadmin approves, the email that is sent out goes to /login, and the user is able to login.
I seem to be missing the redirect where does this occur?

Reply:
@aindriu-aiven the page userActivation is referred here in TemplateController 
https://github.com/Aiven-Open/klaw/blob/main/core/src/main/java/io/aiven/klaw/controller/TemplateMapController.java#L195